### PR TITLE
master-lps-121381

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-121381

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: portal-web/test/functional/com/liferay/portalweb/macros/Questions.macro [hunk 1]
```diff
@@ -562,6 +592,34 @@ definition {
...
+				locator1 = "Questions#VOTE_BUTTON");
+		}
+
+		AssertVisible(locator1 = "Questions#QUESTIONS_BELL_BUTTON");
+
+		AssertVisible(locator1 = "Icon#MOVE_TO_RECYCLE_BIN");
+
+		AssertVisible(locator1 = "Button#EDIT");
+
+		AssertVisible(locator1 = "Button#REPLY");
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/Questions.macro:612]: @Jeremy-Ch when you have redundant assertions you could turn this into a list. That way you can write cleaner code

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/liferay-lima/liferay-portal/pull/883<br></br>

**Pull request closed without merging**